README.md

```diff
@@ -152,6 +154,7 @@ Please [discover modm's peripheral drivers for your specific device][discover].
 <td align="center">✅</td>
 <td align="center">✗</td>
 <td align="center">✗</td>
+<td align="center">✗</td>
```

Comment:
A lot of information in this table is wrong, primarily due to missing drivers making the driver name guessing algorithm to fail. Do we want to extend mapping around `generate_hal_matrix.py:119` now to show all the "present, but not supported" blanks or wait until the drivers gets written?

Reply:
Yeah, you're right 😒. I would like the table to be correct and show the real peripherals, so perhaps you can play around with [the manual additions and see if that improves the table](https://github.com/modm-io/modm/blob/develop/tools/scripts/generate_hal_matrix.py#L118-L128).

Reply:
I fixed most of this. I’ll try to figure out the rest maybe tomorrow.